packages/amplify-provider-awscloudformation/src/iterative-deployment/deployment-manager.ts

```diff
@@ -46,39 +56,46 @@ export class DeploymentError extends Error {
 }
 
 /**
- *
+ * Deployment operation type
  */
 export type DeploymentOp = Omit<DeploymentMachineOp, 'region' | 'stackTemplatePath' | 'stackTemplateUrl'> & {
   stackTemplatePathOrUrl: string;
 };
 
 /**
- *
+ * Deployment step type
  */
 export type DeploymentStep = {
   deployment: DeploymentOp;
   rollback: DeploymentOp;
 };
 
+type EventMap = {
+  rootStackName: string;
+  envName: string;
+  projectName: string;
+  rootResources: {key: string, category: string}[];
+  eventToCategories: Map<string, string>;
+  categories: {name: string, size: number}[];
+};
+
 /**
- *
+ * DeploymentManager class
  */
 export class DeploymentManager {
   /**
    * Helper method to get an instance of the Deployment manager with the right credentials
    */
-
   public static createInstance = async (
     context: $TSContext,
     deploymentBucket: string,
-    spinner: ora.Ora,
+    eventMap: EventMap,
```

Comment:
do export this function from this package?

Reply:
No